src/storage/aggregator.cc

```diff
@@ -786,82 +821,128 @@ bool MinMaxBaseAggregator::DecodeAggrVal(const int8_t* row_ptr, AggrBuffer* buff
     return true;
 }
 
+bool MinMaxBaseAggregator::UpdateAggrVal(const codec::RowView& row_view, const int8_t* row_ptr, AggrBuffer* aggr_buffer,
+                   const std::string& key, bool reverse) {
+    return true;
+}
+
+bool MinMaxBaseAggregator::RebuildExtremumAggrBuffer(const codec::RowView& row_view,
+                                                     AggrBuffer* aggr_buffer,
+                                                     const std::string& key) {
+    if (base_table_ == nullptr) {
+        PDLOG(ERROR, "base table is nullptr, cannot update MinAggr table");
+        return false;
+    }
+    auto it = base_table_->NewTraverseIterator(this->GetIndexPos());
+    int64_t ts_begin = aggr_buffer->ts_begin_;
+    int64_t ts_end = aggr_buffer->ts_end_;
+    it->Seek(key, ts_end);
+    aggr_buffer->clear();
+    if (this->GetWindowType() == WindowType::kRowsRange) {
```

Comment:
for both `kRowsRange` and `kkRowsNum`, the `ts_begin_` and `ts_end_` should keep the same.